src/main/java/com/mongodb/hibernate/internal/translate/mongoast/filter/AstNorFilter.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2025-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.hibernate.internal.translate.mongoast.filter;
+
+import static java.util.Collections.singletonList;
+
+import java.util.List;
+
+public class AstNorFilter extends AbstractAstLogicalFilter {
+
+    public AstNorFilter(AstFilter filter) {
+        this(singletonList(filter));
+    }
+
+    public AstNorFilter(List<? extends AstFilter> filters) {
+        super("$nor", filters);
```

Comment:
We created this constructor, but we don't use it, and its unclear to me if we are going to use it in the future. We generally should not introduce internal API in a PR that does not use it, unless there is a good reason for doing so, but we keep doing that in different PRs.

1. Let's remove the constructor and update `AstNorFilterTests` such that it tests the other constructor?
2. Let's use `AstTestUtils.createFieldOperationFilter` in `AstNorFilterTests`, like it was done in `AstAndFilterTests`, `AstOrFilterTests`.

Reply:
done.